pkg/virt-controller/services/template.go

```diff
@@ -718,6 +718,21 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, t
 		resources.Limits[key] = value
 	}
 
+	// Add ephemeral storage request to container to be used by Kubevirt. This amount of ephemeral storage
+	// should be added to the user's request.
+	ephemeralStorageAddition := resource.MustParse("50M")
+	if ephemeralStorageRequested, ephemeralStorageDefined := resources.Requests[k8sv1.ResourceEphemeralStorage]; ephemeralStorageDefined {
+		ephemeralStorageRequested.Add(ephemeralStorageAddition)
+		resources.Requests[k8sv1.ResourceEphemeralStorage] = ephemeralStorageRequested
+	} else {
+		resources.Requests[k8sv1.ResourceEphemeralStorage] = ephemeralStorageAddition
+	}
```

Comment:
I think this can simply be:
```
ephemeralStorageRequested := resources.Requests[k8sv1.ResourceEphemeralStorage]
ephemeralStorageRequested.Add(ephemeralStorageAddition)
resources.Requests[k8sv1.ResourceEphemeralStorage] = ephemeralStorageRequested
```

if the resources.Requests map doesn't have this key it will return just resource.Quantity{} and then you can simply add your overhead to it...

Reply:
Cool!